website/src/index.html

```diff
@@ -26,9 +26,16 @@ <h1><img src='{{ baseurl }}/images/logo-white.png'></h1>
     <li>Please follow the <a href='https://www.python.org/psf/conduct/'>Code of Conduct</a> at all times.</li>
     <li>We propose talks and workshops in <a href='https://github.com/PyJaipur/PyJaipur/'>our github repo</a></li>
     <li>Our meetups are announced via <a href='https://www.meetup.com/PyJaipur/'>meetup.com</a></li>
+<<<<<<< HEAD
```

Comment:
do we want this stuff here?

Reply:
Instead of doing git pulls I like to do the following:

```bash
cd PyJaipur
Gsync pyj master
# make my changes
Gac
git push origin master
```

Relevant commands can be found at arjoonn.com/static/bashrc